src/magic_folder/uploader.py

```diff
@@ -211,6 +215,7 @@ def add_file(self, path):
 
         # add file into the queue
         d = Deferred()
+        self._status.upload_started()
```

Comment:
If we shutdown while we have local snapshots, I think we'll start uploading them without ever notifying status?

Reply:
yeah .. probably need an additional call in the `RemoteSnapshotCreator`